tests/unit/components/ui/formElements/numberfield.spec.js

```diff
@@ -6,7 +6,7 @@ const localVue = createLocalVue();
 
 beforeEach(() => {
   wrapper = shallowMount(NumberField, {
-    localVue
+    localVue,
```

Comment:
Do the test actually work. Otherwise throw them away, just adds to technical depth, i would rather start fresh if we are going to add tests

Reply:
When running the test commands as stated in the package.json they pass and the server is set up, do not know if they just get skipped or something like that